lib/redis_memo/memoize_query/cached_select.rb

```diff
@@ -111,7 +111,7 @@ def self.install(connection)
           sql.gsub(/(\$\d+)/, '?')      # $1 -> ?
              .gsub(/((, *)*\?)+/, '?')  # (?, ?, ? ...) -> (?)
         end,
-      ) do |_, sql, name, binds, **kwargs|
+      ) do |_, sql, _, binds, **|
```

Comment:
This is just lint but doesn't change functionality?

Reply:
Yep doesn't change the functionality of the memoized method, but changes how we locally cache extracted dependencies, since we will explicitly ignore `nil` or `:_` values in `RedisMemo::MemoizeMethod.map_depends_on_method_args`